test/CadlRanchProjects.Tests/encode-bytes.cs

```diff
@@ -161,5 +160,43 @@ public class EncodeBytesTests : CadlRanchTestBase
             Response response = await new BytesClient(host, null).GetRequestBodyClient().Base64urlAsync(data);
             Assert.AreEqual(204, response.Status);
         });
+
+        [Test]
+        public Task Encode_Bytes_ResponseBody_default() => Test(async (host) =>
+        {
+            var response = await new BytesClient(host, null).GetResponseBodyClient().DefaultAsync();
```

Comment:
The operation return Bytes, not Bytes Array.
You need to check the value, not convert it to array and check.

And all following tests have the same problem.

Reply:
this method is returning `Response<BinaryData>` therefore the result here is a `BinaryData`.

We use `ToArray` to verify the content of two `BinaryData` instances. The other approach is using `ToString`, these are identical.

What is your point?

Reply:
Sorry. I made a mistake about the method `toArray`. Yes, we need to convert `BinaryData` to byte array and then compare.